code-reviews/pull-request-template.md

```diff
@@ -0,0 +1,28 @@
+### Link to the design document that prompted this PR?
+- Provide a link to the design document relevant to this area. 
+
+### What are you trying to address ?
+-  Describe the current behavior that you are modifying, and link to issue number
+
+### Description of new changes
+- Write a detailed description of all changes
+- If you have only done mechanical (not logical) changes point that out here
+-  Describe the new behavior that is introduces
+- If this introduces a breaking change, please describe the impact and migration path for existing applications below.
+
+### How did you test it?
+- Besides writing tests with your change, you should state the manual validation that you have done
+  * [ ] Ran it on specific OS
+  * [ ] Ran it using specific test set
+  * [ ] Description of test scenarios that you have tried.
+
+### Any relevant logs or outputs?
+
+- If you are printing something show a screenshot 
+- When you want to share long logs upload to:
+<br>(StorageAccount)/pr-support/attachments/(PR Number)/(yourFiles) using [Azure Storage Explorer](https://azure.microsoft.com/en-us/features/storage-explorer/) or [portal.azure.com](portal.azure.com) and insert the link here.
+
+
+### Other information or known dependencies
+- Any other information or known dependencies that is important to this PR.
+- ToDos that are to be done after this PR.
```

Comment:
Nit:  I like the way you wrote your TODO with capital letters in the PR for this, should this also be capital letters.